test/scenarios-periodics/el92-prel@el94-crel@optional-upgrade.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+	prepare_kickstart host1 kickstart.ks.template rhel-9.2-microshift-crel-optionals
+	launch_vm host1
```

Comment:
```suggestion
	launch_vm host1 rhel-9.2
```
+ Change `DEFAULT_BOOT_BLUEPRINT` to use `rhel-9.4` in scenario.sh